roadmap/2022.md

```diff
@@ -0,0 +1,57 @@
+# Roadmap 2022
+
+This is the roadmap planned by the Integration team at MeiliSearch for the year 2022.
+
+Nothing is definitive: the roadmap can be adapted depending on the need of the users and the company.
+
+## 2022: the Main Lines
+
+#### Projects
+
+- Better know our users and our community to ease our decision making
```

Comment:
with the help of a PM?

Reply:
Nope, by doing both line below:
- set the analytics on every integration
- Extract information from them to prioritize our tasks